app/controllers/device_story_comments_controller.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+class DeviceStoryCommentsController < ApplicationController
+  before_action :set_device_story
+  before_action :set_device_story_comment, only: %i(show edit update destroy)
+
+  def index
+    @device_story_comments = @device_story.device_story_comments
+  end
+
+  def new
+    @device_story_comment = @device_story.device_story_comments.build
+  end
+
+  def create
+    @device_story_comment = @device_story.device_story_comments.build(device_story_comment_params)
+    respond_to do |format|
+      if @device_story_comment.save && !@device_story_comment.spam?
+        format.html { redirect_to device_story_path(@device_story), notice: 'Comment successfully submitted!' }
+      else
+        flash[:error] = 'Could not submit your comment. Please, try again!'
+        format.html { redirect_to device_story_path(@device_story) }
+      end
+    end
+  end
+
+  def update
+    respond_to do |format|
+      if @device_story_comment.update(device_story_comment_params)
+        format.html { redirect_to device_story_path(@device_story), notice: 'device_story_comment was successfully updated.' }
+        format.json { render :show, status: :ok, location: @device_story_comment }
+      else
+        format.html { render :edit }
+        format.json { render json: @device_story_comment.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  def destroy
+    @device_story_comment.destroy
```

Comment:
Pretty sure at the moment this allows any user to delete (or update) any comment, we should set it so only moderator or comment author can do that.